### ISSUE TITLE
writevtk for empty triangulations

### ISSUE THREAD

**Issue by fverdugo** [gridap/Gridap.jl#762]
This does not work:
```julia
using Gridap
cells = (10,10)
domain = (0,1,0,1)
model = CartesianDiscreteModel(domain,cells)
Ω0 = Interior(model,Int[])
writevtk(Ω0,"empty_mesh")
```

We need this, e.g. in parallel computations when some subdomain is empty.

**Comment by fverdugo** [comment]
@amartinhuertas @ericneiva @oriolcg , someone volunteering to fix this?

**Comment by fverdugo** [comment]
WIP in #766

**Comment by fverdugo** [comment]
closed via #766
